Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/certificateselection/DccTicketingCertificateSelectionHelperFunctions.kt

```diff
@@ -0,0 +1,27 @@
+package de.rki.coronawarnapp.dccticketing.ui.certificateselection
+
+import android.content.Context
+import de.rki.coronawarnapp.R
+
+fun getRequestedCertificateTypes(requestedCertificatesList: List<String>, context: Context, separator: String): String =
+    requestedCertificatesList.joinToString(separator) { certificate ->
+        when (certificate) {
+            FilterType.VACCINATION.type -> context.getString(R.string.vaccination_certificate_name)
+            FilterType.RECOVERY.type -> context.getString(R.string.recovery_certificate_name)
+            FilterType.PCR_TEST.type,
+            FilterType.TEST.type,
+            FilterType.RA_TEST.type -> context.getString(R.string.test_certificate_name)
+            else -> ""
+        }
+    }
+
+fun getFullName(familyName: String?, givenName: String?): String =
+    listOf(familyName, givenName).joinToString("<<")
+
+private enum class FilterType(val type: String) {
```

Comment:
We have one already in case you want to use please use the same one , maybe you move into an enum class and can be used in both cases ?

Reply:
Yep, fair enough will make an enum class for both cases.